src/Leela.cpp

```diff
@@ -258,6 +266,17 @@ static void parse_commandline(int argc, char *argv[]) {
     }
 #endif
 
+    if (vm.count("benchmark")) {
+        cfg_noise = false;
+        cfg_random_cnt = 0;
+        cfg_timemanage = TimeManagement::OFF;
+        if (!vm.count("playouts") && !vm.count("visits")) {
+            // Default to self-play and match values.
+            cfg_max_playouts = 1600;
+            cfg_allow_pondering = false;
```

Comment:
Pondering should probably always be disabled during a bench.

Reply:
Done.